pysb/core.py

```diff
@@ -153,6 +153,16 @@ def rename(obj, new_name):
                              "name '%s'" % obj.name)
 
 
+class Symbol(sympy.Dummy):
+    def __new__(cls, name, value=0.0, _export=True):
```

Comment:
Could the `value` and `_export` args be removed here?

Reply:
yes! removed.